sockeye/inference.py

```diff
@@ -1369,3 +1424,30 @@ def _get_best_from_beam(self,
             else:
                 result.append(Translation(sequence, attention_matrix, score))
         return result
+
+    def _print_beam(self,
```

Comment:
this function should be rewritten to use log.debug statements or be completely removed.